source/endpoints/transcript.rst

```diff
@@ -0,0 +1,112 @@
+*************
+Transcript
+*************
+
+``GET /jobs/{id}/transcript``
+******************************
+
+Gets an existing transcription job by id
+
+Note: all GET transcript requests must include an ``Accept`` header that specifies the desired output format. 
+
+**CURL Examples**
+
+.. code:: javascript
+
+  curl -X GET "https://api.rev.ai/revspeech/v1beta/jobs/{id}" -H "Authorization: Bearer <api_key>"
+
+**Parameters**
+
+====================== ===============================================================
+Name                   Description
+====================== ===============================================================
+id ``(required)``      Gets an existing transcription by job id
+====================== ===============================================================
+
+**Responses**
+
+====================== ===============================================================
+Code                   Description
+====================== ===============================================================
+200                    Revspeech API Transcript
+
+                       ``Example Value``
+
+                       **Accept: application/vnd.rev.transcript.v1.0+json**
+
+                       .. code:: javascript
+
+                        {
+                          "monologues": [
+                            {
+                              "speaker": 1,
+                              "elements": [
+                                {
+                                  "type": "text",
+                                  "value": "Hello",
+                                  "ts": 0.5,
```

Comment:
we should probably document these fields

Reply:
This is all documented in the models?